$NetBSD: patch-tests_Makefile.am,v 1.1 2016/08/11 03:23:44 mef Exp $

Disable bison tests since we don't want to build with bison
to avoid a dependency cycle, since bison depends on flex.

 bison_yylloc_main.$(OBJEXT): bison_yylloc_parser.h bison_yylloc_scanner.h
--- tests/Makefile.am.orig	2016-03-02 09:24:24.000000000 +0900
+++ tests/Makefile.am	2016-08-11 12:17:05.000000000 +0900
@@ -64,9 +64,6 @@ simple_tests = \
 	array_r \
 	basic_nr \
 	basic_r \
-	bison_nr \
-	bison_yylloc \
-	bison_yylval \
 	c_cxx_nr \
 	c_cxx_r \
 	ccl \
@@ -170,12 +167,6 @@ top_SOURCES = top.l top_main.c
 yyextra_SOURCES = yyextra.l
 
 BUILT_SOURCES = \
-	bison_nr_parser.h \
-	bison_nr_scanner.h \
-	bison_yylloc_parser.h \
-	bison_yylloc_scanner.h \
-	bison_yylval_parser.h \
-	bison_yylval_scanner.h \
 	header_nr_scanner.h \
 	header_r_scanner.h \
 	multiple_scanners_nr_1.h \
